try:
    print(x)
except:
    print("Ocurrio un error")    
    